pulp_rpm/app/tasks/synchronizing.py

```diff
@@ -571,6 +561,10 @@ def is_subrepo(directory):
         with RpmPublication.create(
             repo_sync_results[PRIMARY_REPO], pass_through=False
         ) as publication:
+            # TODO: how do we create a publication if we have no "root" repository version? we need to pick one... it isn't obvious which one
+            # Or can we pick an empty / the previous repository version of the current repo?"
+
+            # TODO: no new repository version == no autopublish. that's kind of fundamental to how autopublish works currently, I'm not sure how to work around it
```

Comment:
@ipanova @ggainey

What do you think the most reasonable answers are, here.  This second question I'm not even sure we can fix without a rework of autopublish which would fall under a separate issue.